test/test.js

```diff
@@ -0,0 +1,212 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const axios = require('axios');
+const chai = require('chai');
+const config = require('./config');
+const fs = require('fs');
+const helpers = require('./helpers');
+const https = require('https');
+const mockData = require('./mock-data');
+const mockServer = require('./mock-server');
+const should = chai.should();
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
+
+let server = null;
+
+describe('http-signature-middleware', () => {
+  before(() => {
+    const key = fs.readFileSync('./test/basic-server.key');
+    const cert = fs.readFileSync('./test/basic-server.crt');
+    server = https.createServer({cert, key}, mockServer.createServer());
+    server.listen(config.port);
+  });
+
+  after(() => {
+    server.close();
+  });
+
+  it('dereferences an https ed25519 key', async() => {
+    const requestOptions = {
+      headers: {},
+      method: 'GET',
+      url: `https://localhost:${config.port}/alpha`,
+    };
+    const identity = mockData.identities.alpha;
+    await helpers.createHttpSignatureRequest(
+      {algorithm: 'ed25519', identity, requestOptions});
+    let response;
+    try {
+      response = await axios.get(requestOptions.url, requestOptions)
```

Comment:
Can simplify all these calls with method and url in the options object to just `await axios(requestOptions)`.

Reply:
done.